.github/workflows/config/spelling_allowlist.txt

```diff
@@ -198,10 +202,13 @@ lossy
 lvalue
 makefiles
 merchantability
+mlir
 mutex
 namespace
 namespaces
 natively
+ncols
```

Comment:
Use backticks `ncols` or spell out as `number of columns`

Reply:
Fixed in [a3964e2](https://github.com/NVIDIA/cuda-quantum/pull/1763/commits/a3964e284910f99cbbddddfa7493747ab9735855).